phantomjs/bridge.js

```diff
@@ -9,7 +9,7 @@
 /*global QUnit:true, alert:true*/
 (function (factory) {
   if (typeof define === 'function' && define.amd) {
-    require(['qunit'], factory);
+    define('phantom-bridge', ['qunit'], factory);
```

Comment:
I have little experience with `RequireJS`, I just re-read the documentation about named modules and copy-pasta their recommendation.

The other links are some of the solutions used to make the three components works together, because the bridge file is always injected, unless you created yours and require it using `RequireJS`.

In the latest case, you have to set `inject` option to empty string in order to prevent the auto injection of the built-in bridge.